<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurveyInitDialogBase</class>
 <widget class="QDialog" name="SurveyInitDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="mSurveyAreaLayerLabel">
       <property name="text">
        <string>Survey area layer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mSurveyAreaLayerComboBox"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="mNewSurveyLayerButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/plugins/survey_design/mActionNewVectorLayer.png</normaloff>:/plugins/survey_design/mActionNewVectorLayer.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mBaselineLayerLabel">
       <property name="text">
        <string>Survey baseline layer</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="mSurveyBaselineLayerComboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="mNewBaselineLayerButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/plugins/survey_design/mActionNewVectorLayer.png</normaloff>:/plugins/survey_design/mActionNewVectorLayer.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="mStrataGroupBox">
     <property name="title">
      <string>Strata</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mStrataLayerLabel">
        <property name="text">
         <string>Strata layer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mStrataLayerLabelComboBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="mNewStrataLayerButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/plugins/survey_design/mActionNewVectorLayer.png</normaloff>:/plugins/survey_design/mActionNewVectorLayer.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mMinDistanceLabel">
        <property name="text">
         <string>Minimum distance attribute</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mMinimumDistanceAttributeComboBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/plugins/survey_design/mActionNewAttribute.png</normaloff>:/plugins/survey_design/mActionNewAttribute.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mNumberOfSamplePointsAttribute">
        <property name="text">
         <string>N sample points attribute</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="toolButton_2">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/plugins/survey_design/mActionNewAttribute.png</normaloff>:/plugins/survey_design/mActionNewAttribute.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SurveyInitDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SurveyInitDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
